web/docs/Backend Starters/Decoupled Drupal/configuring-build-hooks.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-drupal/configuring-build-hooks'
+sidebar_position: 5
+---
+
+## Setting Up Your First Build Hook
+
+### What Are Build Hooks?
+
+The Build Hooks module, available at https://www.drupal.org/project/build_hooks,
+allows CMS events like creating, updating, or deleting content to trigger builds
+in a decoupled frontend site.
+
+### Installing the Build Hooks Module
+
+To add Build Hooks as a composer dependency, run the following command:
+
+```
+composer require 'drupal/build_hooks:^3.3'
+```
+
+### Creating a Build Hook for Content Creation/Modification
```

Comment:
It is more verbose, but I think it will be more useful to split this out into two full sections:

Creating a Build Hook Triggered Automatically When Content Changes

Creating a Build Hook Triggered Manually

Each will include the steps for picking a deployment strategy and testing (probably not as their own sections) Many of the instructions will be the same, but for the manual one, we can go into more detail about where the user will need to go in the Drupal admin, and how they can trigger the hook manually.

I don't think we need a full Cron example, but maybe we could note that deployment strategy is also an option.